<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableTab</class>
 <widget class="QDialog" name="TableTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Table #</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time played</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time in $</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>F&amp;B $</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeInDollars">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="FoodNBeverage">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TimePlayed">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TableId">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TOTAL</string>
   </property>
  </widget>
  <widget class="QLabel" name="Total">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(253, 124, 53);</string>
   </property>
   <property name="text">
    <string>   Settle the check before closing table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseTable_Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Table</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
